girder/girder_large_image/web_client/vue/components/FrameSelector.vue

```diff
@@ -172,10 +174,27 @@ export default Vue.extend({
             />
         </table>
 
-        <div v-if="currentModeId > 1" class="image-frame-simple-control">
+        <!-- Use composite layers component twice so state for each one is maintained while invisible -->
+        <!-- Use styling instead of v-if to make each invisible so that the components are not unmounted -->
+        <div  class="image-frame-simple-control" style="width: 500px">
```

Comment:
Please don't use inline styles -- they are hard to override when we reuse the component elsewhere  (like in HistomicsUI)

Reply:
Oh thanks, I hadn't meant to leave that in there. I wanted to see how it would look when squished for HistomicsUI. Removed in [4f10794](https://github.com/girder/large_image/pull/1175/commits/4f10794f9e22a0bf795e7c544b2af064834579fd)